### DISCUSSION TITLE
修改获取的数据数量

### DISCUSSION THREAD

**Discussion by ZhaoZiqi7** [KittenCN/stock_prediction#19] [General]
请问为什么修改getdata中调用的get_stock_data函数，修改datediff并无效果

**Comment by KittenCN** [comment]
datediff是表示结束日期是当前前多少天,应该是有作用的

**Reply by ZhaoZiqi7** [reply]
目前的情况是，我先尝试成功运行您的代码，发现数据量过多，但是无论我如何修改datediff，始终都是5388个，很奇怪

**Reply by KittenCN** [reply]
5388表示的是有5388只股票，datediff表示的是时间差，开始时间永远是该股票的开票第一天，如果想只盯着某一只股票，就把股票代码写到引号中间

**Reply by ZhaoZiqi7** [reply]
多谢多谢 我再试试

**Reply by ZhaoZiqi7** [reply]
请问为什么出现了这个 但是没有出现图片哇
![image](https://github.com/KittenCN/stock_prediction/assets/127832717/0e992ec2-40b4-492b-b8f2-2bdd586a3302)

**Reply by KittenCN** [reply]
test结果在png目录下，如果是test就在test目录，如果是训练结束就在train_loss目录

**Reply by ZhaoZiqi7** [reply]
多谢多谢 麻烦您了
